packages/core/src/components/DragOverlay/DragOverlay.tsx

```diff
@@ -88,6 +89,7 @@ export const DragOverlay = React.memo(
         };
     const style: React.CSSProperties | undefined = activeNodeRect
       ? {
+          ...propStyle,
```

Comment:
I think it would be better to move `...propStyle,` down to the end of `style`.

If we are giving users the ability to control the style on such a low level, I think they should have full control of the prop, so they can override styles if needed.

Just my two cents. Thank you @inokawa for taking the initiative!! I had this problem and appreciate you fixing this.

Reply:
I agree with you. Thank you!
I fixed in https://github.com/clauderic/dnd-kit/pull/246/commits/b9b0f73e02eb7ee32f7da15ff90548dbd1878efc